lib/ui/widgets/empty_page.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+import '../../common/constants.dart';
+
+class EmptyPage extends StatelessWidget {
+  final String vectorImagePath;
+  final String title;
+  final String description;
+
+  const EmptyPage({
+    super.key,
+    required this.vectorImagePath,
+    required this.title,
+    required this.description,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Padding(
+        padding: const EdgeInsets.all(GapSize.xl),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            SvgPicture.asset(
+              vectorImagePath,
+              height: 250,
+              fit: BoxFit.cover,
+            ),
+            const SizedBox(height: GapSize.xl),
+            Text(
+              title,
+              style: Theme.of(context).textTheme.titleLarge,
+              textAlign: TextAlign.center,
+            ),
+            const SizedBox(height: GapSize.m),
```

Comment:
Nit: The gap is a bit large (24 in app vs 16 in figma)

Reply:
Fixed, thanks for your watchful eyes